bin/circleci.sh

```diff
@@ -3,6 +3,7 @@ export DOCKER_VERSION=${1:-1.9.1}
 export BUILD_HARNESS_PROJECT=${2:-build-harness}
 export BUILD_HARNESS_BRANCH=${3:-master}
 export GITHUB_REPO="git@github.com:sagansystems/${BUILD_HARNESS_PROJECT}.git"
+export PATH="$PATH:/usr/local/bin"
```

Comment:
it probably is; i'll remove it. i assumed this is why the `rc` files were being sourced, so I added this. =)